packages/wdio-cli/src/commands/config.ts

```diff
@@ -269,7 +275,11 @@ export async function missingConfigurationPrompt(command: string, message: strin
      */
     if (!config && !process.env.VITEST_WORKER_ID) {
         /* istanbul ignore next */
-        console.log(message)
+        if (command === 'run'){
+            console.log(`No WebdriverIO configuration found in "${process.cwd()}"`)
+        } else {
+            console.log(message)
+        }
```

Comment:
```suggestion
        console.log(command === 'run'
            ? `No WebdriverIO configuration found in "${process.cwd()}"`
            : message
```

Reply:
@christian-bromann Thanks for the suggestions. Committed with the changes you had suggested and also the tests are passing in local now.